.github/workflows/docs-preview.yml

```diff
@@ -53,7 +53,7 @@ jobs:
 
       ### Build old docs
       - name: Build docs
-        run: SPHINXOPTS="-W" make -C docs build
+        run: SPHINXOPTS="-W" make -C docs deps-test build
```

Comment:
What changed that meant we needed to install these here?

Reply:
#741 

I'm loathe to spend too much time thinking about this, since we'll be deleting almost the entirety of this directory in a few weeks time.

Reply:
Which is me acknowledging this approach isn't great, but enough for now(?)

Reply:
Ok didn't realize this was building on something probably @plotnick should review since he reviewed that but also agreed this doesn't seem too important.